docs/API.md

```diff
@@ -160,8 +160,8 @@ This parameter is a string that are an IPv4 or IPv6. It's validated with the fol
 
 Basic data type: string
 
-A string of A-Z, a-z and underscores matching the regular expression
-`/^[a-z]{2}(_[A-Z]{2})?$/`.
+A string matching the regular expression `/^[a-z]{2}$/` (preferred format) or a string 
+matching the regular expressions `/^[a-z]{2}_[A-Z]{2}$/` (*deprecated* format).
```

Comment:
I'd like to see the deprecation highlighted more strongly here at the very beginning of the Language tag definition to make sure people notice it as soon as they locate the Language tag heading. What do you think of something like this?

> A string matching one of the following regular expressions:
> * `/^[a-z]{2}$/` (preferred format)
> * `/^[a-z]{2}_[A-Z]{2}$/` **Deprecated**: Use the preferred format instead.

Reply:
I will do the update.